model/common_structures.smithy

```diff
@@ -59,3 +59,90 @@ structure Hits{
     _source: Document,
     fields: Document
 }
+
+structure Audit{
+    _readonly: ReadOnlyList
+    config: Config
+}
+
+structure Config{
+    compliance: Compliance
+    enabled: Boolean
+    audit: AuditLogs
+}
+
+list ReadOnlyList{
+    member: String
+}
+
+structure AuditLogs{
+    ignore_users: IgnoreUsersList
+    ignore_requests: IgnoreRequests
+    disabled_rest_categories: DisabledRestCategories
+    disabled_transport_categories: DisabledTransportCategories
+    log_request_body: Boolean
+    resolve_indices: Boolean
+    resolve_bulk_requests: Boolean
+    exclude_sensitive_headers: Boolean
+    enable_transport: Boolean
+    enable_rest: Boolean
+}
+
+structure Compliance{
+    enabled: Boolean
+    write_log_diffs: Boolean
+    read_watched_fields: AttributeMap
+    read_ignore_users: IgnoreUsersList
+    write_watched_indices: WriteWatchedIndices
+    write_ignore_users: WriteIgnoreUsers
+    read_metadata_only: Boolean
+    write_metadata_only: Boolean
+    external_config: Boolean
+    internal_config: Boolean
+}
+
+map AttributeMap {
+    key: String,
+    value: String
+}
+
+list IgnoreUsersList{
+    member: String
+}
+
+list IgnoreRequests{
+    member: String
+}
+
+list DisabledRestCategories{
+    member: String
+}
+
+list DisabledTransportCategories{
+    member: String
+}
+
+list WriteWatchedIndices{
+    member: String
+}
+
+list WriteIgnoreUsers{
+    member: String
+}
+
+structure AuditConfig{
+    enabled: Boolean
+    audit: AuditLogs
+    compliance: Compliance
+}
+
+structure AuditResponse{
+    status: String
+    message: String
+}
+
+structure PatchOperation {
+    op: String,
+    path: String,
+    value: AttributeMap
+}
```

Comment:
This file will grown extremely large if we put every structure here. It's only meant for structure to be reused like the results of different types of search. Can we move these into the `structures.smithy` file of the respective operation?

Reply:
I tried to move structures in all PRs that are not used in several APIs. But some will have to be left, it may seem cumbersome, but some PR will coincide with the addition of a map or response. I hope that I understood you correctly, I did the best I could.